survey_runner_jmeter.rb

```diff
@@ -85,66 +90,68 @@ def post_introduction()
 def post_page_1_empty()
     submit name: 'POST page 1 (empty)', url: '${url}',
             fill_in: {
-                "6cf5c72a-c1bf-4d0c-af6c-d0f07bc5b65b":"",
-                "92e49d93-cbdc-4bcb-adb2-0e0af6c9a07c":"",
-                "a5dc09e8-36f2-4bf4-97be-c9e6ca8cbe0d":"",
-                "9587eb9b-f24e-4dc0-ac94-66117b896c10[]":"",
                 "6fd644b0-798e-4a58-a393-a438b32fe637-day":"",
                 "6fd644b0-798e-4a58-a393-a438b32fe637-month":"",
                 "6fd644b0-798e-4a58-a393-a438b32fe637-year":"",
                 "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0-day":"",
                 "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0-month":"",
                 "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0-year":"",
                 "action[save_continue]": "" } do
-        assert contains: ['This field is mandatory.', 'date entered is not valid'], scope: 'main'
+        assert contains: ['These must be corrected to continue.', 'date entered is not valid'], scope: 'main'
         extract_url
     end
 end
 
 def post_page_1_filled()
     submit name: 'POST page 1 (filled)', url: '${url}',
             fill_in: {
-                "6cf5c72a-c1bf-4d0c-af6c-d0f07bc5b65b":"99",
-                "92e49d93-cbdc-4bcb-adb2-0e0af6c9a07c":"1024",
-                "a5dc09e8-36f2-4bf4-97be-c9e6ca8cbe0d":"Lion",
-                "9587eb9b-f24e-4dc0-ac94-66117b896c10[]":"Yoda",
+                # sales period
```

Comment:
Good job adding comments in to describe what each field is.

Debatable point: I don't know why we don't give the ID fields a more reflective name as long as they are unique. Why not have the id as 'from_sales_period'